metadata.yaml

```diff
@@ -0,0 +1,38 @@
+name: sdcore-smf-operator
```

Comment:
Please add a `CODEOWNERS` file with the following content:
```
*       @canonical/telco
```
This will automatically tag canonical/telco team for PR's.

Reply:
Added, and nice feature BTW